src/codeu/chat/client/core/Context.java

```diff
@@ -47,4 +47,9 @@ public UserContext create(String name) {
     }
     return users;
   }
+
+  //Creating a basic context with no information: users, conversations or messages
+  public UptimeServerInfo getInfo(){
+      return view.getInfo();
+  }
```

Comment:
Because VersionCheck and UpTime have different ServerInfo classes, I believe you would need a getInfo for VersionServerInfo as well.